app/src/main/java/org/mozilla/fenix/settings/advanced/LocaleManagerExtension.kt

```diff
@@ -51,3 +51,15 @@ fun LocaleManager.getSelectedLocale(
 fun LocaleManager.isDefaultLocaleSelected(context: Context): Boolean {
     return getCurrentLocale(context) == null
 }
+
+/**
+ * Update the locale for the configuration of the app context's resources
+ */
+@Suppress("DEPRECATION")
```

Comment:
This is a workaround, since Android does not support in-app locale changes. [I added a task in AC to search for a long-term solution](https://github.com/mozilla-mobile/android-components/issues/5631)